src/test/java/Page_Objects/SongListPage.java

```diff
@@ -124,4 +126,30 @@ public void approveSongByKeyboard() {
     public boolean isSongMarkedWithDuetIcon(String songID) {
         return getSongElement(songID).findElement(By.cssSelector("[data-test='multitrack-indicator']")).isDisplayed();
     }
+
+    public void expectSongToBeMarkedAsPlayedToday(String songID) {
+        String playedStatusText = getSongElement(songID).findElement(By.cssSelector("[data-test='song-stat-indicator']")).getText();
+        String playedStatusSubs = playedStatusText.substring(0, 1).toUpperCase() + playedStatusText.substring(1).toLowerCase();

```

Comment:
why not just put everything toLowerCase, would be simpler

Reply:
good point, thanks :)